### PR TITLE
Identity provider

### PR DESCRIPTION
**Pull request by kasungayan**

### CODE REVIEW

File: components/openid/org.wso2.carbon.identity.provider/src/main/java/org/wso2/carbon/identity/provider/openid/handlers/OpenIDHandler.java [hunk 1]
```diff
@@ -689,19 +690,19 @@ private void sendToApprovalPage(HttpServletRequest req, HttpServletResponse resp
         Map<ClaimMapping, String> userAttributes = authnResult.getSubject().getUserAttributes();
 
         out.println("<input type='hidden' name='openid.identity' value='" +
-                    session.getAttribute(OpenIDConstants.SessionAttribute.AUTHENTICATED_OPENID) + "'>");
+                session.getAttribute(OpenIDConstants.SessionAttribute.AUTHENTICATED_OPENID) + "'>");
```
Comment (malithie) [inline components/openid/org.wso2.carbon.identity.provider/src/main/java/org/wso2/carbon/identity/provider/openid/handlers/OpenIDHandler.java]: Should encode for html attribute here

File: components/openid/org.wso2.carbon.identity.provider/src/main/java/org/wso2/carbon/identity/provider/openid/handlers/OpenIDHandler.java [hunk 2]
```diff
@@ -689,19 +690,19 @@ private void sendToApprovalPage(HttpServletRequest req, HttpServletResponse resp
...
         out.println("<input type='hidden' name='openid.return_to' value='" +
-                    ((ParameterList) session.getAttribute(OpenId.PARAM_LIST)).getParameterValue(OpenId.ATTR_RETURN_TO) +
-                    "'>");
+                ((ParameterList) session.getAttribute(OpenId.PARAM_LIST)).getParameterValue(OpenId.ATTR_RETURN_TO) +
```
Comment (malithie) [inline components/openid/org.wso2.carbon.identity.provider/src/main/java/org/wso2/carbon/identity/provider/openid/handlers/OpenIDHandler.java]: Should encode for html attribute here

**Comment by malithie** [comment]
import package org.owasp.encoder package to the osgi bundle

**Comment by kasungayan** [comment]
Fixed with #1052

**Pull request closed without merging**